partner_contact_sale_info_propagation/tests/test_partner_contact_sale_info_propagation.py

```diff
@@ -1,67 +1,69 @@
 # Copyright 2019 Tecnativa - Ernesto Tejeda
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import TransactionCase
 
-from lxml import etree
 
-from odoo.tests.common import Form, TransactionCase
-
-
-class TestPartnerContactSaleInfoPropagation(TransactionCase):
+class TestResPartner(TransactionCase):
     def setUp(self):
         super().setUp()
         self.partner_model = self.env["res.partner"].with_context(test_propagation=True)
-        self.parent_company = self.partner_model.create(
+        self.salesperson = self.env["res.users"].create(
+            {
+                "name": "Test Salesperson",
+                "login": "sales@test.com",
+            }
+        )
+        self.team = self.env["crm.team"].create({"name": "Sales Team A"})
+        self.company = self.partner_model.create(
```

Comment:
Don't name it as company, as that's a confusing terminology, thinking that is a `res.company` record. Call it `parent_partner` for example.